src/app/public/modules/open-on-click-directive/open-on-click.directive.spec.ts

```diff
@@ -58,23 +63,34 @@ describe('bbHelpDisableWidget Directive', () => {
   }));
 
   it('should call the widget service open method on enter keypress', fakeAsync(() => {
+    fixture.detectChanges();
+
     let aTag = fixture.debugElement.nativeElement.querySelector('a');
     let openSpy = spyOn(mockWidgetService, 'openWidget').and.callThrough();
-    const enterEvent = new KeyboardEvent('keydown', { key: 'Enter' });
-    fixture.detectChanges();
-    aTag.dispatchEvent(enterEvent);
-    fixture.whenStable().then(() => {
-      expect(openSpy).toHaveBeenCalledWith('foo.html');
+
+    SkyAppTestUtility.fireDomEvent(aTag, 'keydown', {
+      keyboardEventInit: {
+        key: 'Enter'
+      }
     });
+
+    fixture.detectChanges();
+    tick();
+
+    expect(openSpy).toHaveBeenCalledWith('foo.html');
   }));
 
   it('should not call the widget service open method on other keypresses', fakeAsync(() => {
```

Comment:
is this test missing a `tick`? or does it not need `fakeAsync`?

Reply:
Good catch. Fixed!